flask/test/data/qr_code.py

```diff
@@ -6,11 +6,20 @@ def default_qr_code_data():
     mock_qr_code = {"id": 1, "code": "999"}
     return mock_qr_code
 
+def qr_code_without_box_data():
```

Comment:
The structure of the QR code is the same as in `default_qr_code_data`. Why did you not use that one?

Reply:
I think I understood; it's because the default box data has the ID from the default QR code data...?

Reply:
Exactly I wanted to test that the default QR code returns the default QR box. And then I wanted a second QR code not associated with any boxes for the other test case.